docs/user/quickstart/installation.md

```diff
@@ -0,0 +1,126 @@
+(installation)=
+
+*Tutorials written by Christina Ertural ([christina.ertural@bam.de](mailto:christina.ertural@bam.de)).*
+
+# Installation guide
+
+## Before you start using `autoplex`
+
+We expect the general user of `autoplex` to be familiar with the [Materials Project](https://github.com/materialsproject) framework software tools and related 
+packages for (high-throughput) workflow submission and management. 
+This involves the following software packages:
+- [pymatgen](https://github.com/materialsproject/pymatgen) for input and output handling of computational materials science software,
+- [atomate2](https://github.com/materialsproject/atomate2) for providing a library of pre-defined computational materials science workflows,
+- [jobflow](https://github.com/materialsproject/jobflow) for processes, job and workflow handling, 
+- [jobflow-remote](https://github.com/Matgenix/jobflow-remote) or [FireWorks](https://github.com/materialsproject/fireworks) for workflow and database (MongoDB) management,
+- [MongoDB](https://www.mongodb.com/) as the database (we recommend installing the MongoDB community edition). 
+
+## Installation Documentation and Guids of the Dependencies
+
+The first step you need to do is to set up a [MongoDB](https://www.mongodb.com/) database. Help and tips regarding the MongoDB installation 
+can be found [here](https://materialsproject.github.io/fireworks/installation.html#install-mongodb). 
+We recommend installing the [MongoDB community edition](https://www.mongodb.com/docs/manual/administration/install-community/).
+MongoDB also provides lots of [tutorials](https://www.mongodb.com/docs/manual/administration/self-managed-configuration-and-maintenance/)
+to setup and manage your database. Also consider asking your IT administration for help.
+
```

Comment:
Here I also want to include an adjusted version of Aakash's MongoDB setup tutorial ad that would make him a co-author here of course 😃

Reply:
Yes, sure. I am wondering how to recognize the authorship in the best way. I think it will get confusing when we have many new contributors.

Reply:
Do you think we have to track it so detailed? 👀  Personally, I'm fine if we don't do it.

Reply:
Yeah, I don't know. Not too.important. 😅

Reply:
I might get the others opinions too and then we can think about it more 👍🏻

Reply:
Done!